static/opendj/101-ox.ldif

```diff
@@ -2470,6 +2470,24 @@ attributeTypes: ( 1.3.6.1.4.1.48710.1.3.427 NAME 'gluuConfStatic'
   SUBSTR caseIgnoreSubstringsMatch
   SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
   X-ORIGIN 'Gluu created attribute' )
+attributeTypes: ( 1.3.6.1.4.1.48710.1.3.428 NAME 'authReqId'
+  DESC 'Authentication request id'
+  EQUALITY caseIgnoreMatch
+  SUBSTR caseIgnoreSubstringsMatch
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
+  X-ORIGIN 'Gluu created attribute' )
+attributeTypes: ( 1.3.6.1.4.1.48710.1.3.429 NAME 'requestDate'
+  DESC 'Request registration date'
+  EQUALITY generalizedTimeMatch
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.24
+  ORDERING generalizedTimeOrderingMatch
+  X-ORIGIN 'Gluu created attribute' )
+attributeTypes: ( 1.3.6.1.4.1.48710.1.3.430 NAME 'status'
+  DESC 'Status'
+  EQUALITY caseIgnoreMatch
+  SUBSTR caseIgnoreSubstringsMatch
+  SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
+  X-ORIGIN 'Gluu created attribute' )
```

Comment:
There no changes in gluu_schema.json. 101-ox.ldif is generated from gluu_schema.json, so we should put changes there too. Is `requestDate` creation time? If yes, then you can reuse existing creationDate or iat, and for status we have oxStatus already. We should strive avoid duplication, we already have so much of duplicated attributes.

Reply:
Ok, I think I will need to change the code implemented also, I'm going to follow your suggestion. Also if a request should be removed, then I don't need `status` attribute.